docs/source/tutorials/advanced/backbones.rst

```diff
@@ -1,7 +1,142 @@
 
-<Guidelines for writing>
+This tutorial explains how to utilize public backbones.
 
-This tutorial explains how to modify hyper-parameters and configurations, as well as how to utilize public backbones.
+The process has been tested on the following configuration.
+
+- Ubuntu 20.04
+- NVIDIA GeForce RTX 3090
+- Intel(R) Core(TM) i9-11900
+- CUDA Toolkit 11.1, python 3.9
+
+*****************************
+Currently supported backbones
+*****************************
+
+The following libraries are currently available for backbone replacement.
+
++-----------------------+-------+-------+-------------+-----------+-----------+
+|         Task          | mmdet | mmseg | torchvision | pytorchcv | omz.mmcls |
++=======================+=======+=======+=============+===========+===========+
+|    Classification     |   O   |   O   |      O      |     O     |     O     |
++-----------------------+-------+-------+-------------+-----------+-----------+
+|       Detection       |   O   |   O   |      O      |     O     |           |
++-----------------------+-------+-------+-------------+-----------+-----------+
+|     Segmentation      |   O   |   O   |      O      |     O     |           |
++-----------------------+-------+-------+-------------+-----------+-----------+
+| Instance-Segmentation |   O   |   O   |      O      |     O     |           |
++-----------------------+-------+-------+-------------+-----------+-----------+
+
+*************************
+Setup virtual environment
+*************************
+
+You can follow the installation process from a :doc:`quick start guide <../../get_started/installation>` to create a universal virtual environment for OTX.
+
+*****************************
+Backbone replacement tutorial
+*****************************
+
+1. First, we need to configure the workspace for the backbone replacement.
+
+  .. note::
+
+    You can use the OTX workspace to swap out backbones, train, set up configurations, and more.
+    Workspaces are created automatically on otx build or otx train.
+
+  .. code-block::
+
+    (otx) ...$ otx build --task classification
+
+    [*] Workspace Path: otx-workspace-CLASSIFICATION
+    [*] Load Model Template ID: Custom_Image_Classification_EfficinetNet-B0
+    [*] Load Model Name: EfficientNet-B0
+    [*]     - Updated: otx-workspace-CLASSIFICATION/model.py
+    [*]     - Updated: otx-workspace-CLASSIFICATION/data_pipeline.py
+    [*]     - Updated: otx-workspace-CLASSIFICATION/deployment.py
+    [*]     - Updated: otx-workspace-CLASSIFICATION/hpo_config.yaml
+    [*]     - Updated: otx-workspace-CLASSIFICATION/model_hierarchical.py
+    [*]     - Updated: otx-workspace-CLASSIFICATION/model_multilabel.py
+    [*]     - Updated: otx-workspace-CLASSIFICATION/compression_config.json
+    [*] Update data configuration file to: otx-workspace-CLASSIFICATION/data.yaml
+
+    (otx) ...$ cd otx-workspace-CLASSIFICATION
+
+2. Next, we can find the backbone we want to replace via otx find.
```

Comment:
Please, use 2 lines for points. Because it renders with indent.
Instead use: 
"2. Next, we can find the backbone 
we want to replace via otx find."